enterprise/setup/kubernetes_as_non_root.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: 'Enterprise Server: Initial Non-root Setup with Kubernetes'
+backlink_href: /docs/enterprise
+backlink_title: 'pganalyze Enterprise Server'
+---
+
+import ToC from '../../components/Toc'
+
+In some environments, running containers as the root user is not acceptable. This guide describes how to set up [pganalyze Enterprise Server](/enterprise-postgres-monitoring) 
+in a Kubernetes cluster, without running the container as root. This requires using the slim version of the pganalyze Enterprise Docker image and a separate Redis container.
+
+**Please note** that the slim container image does not contain a build in pganalyze collector application. Therefore, you will need to have a standalone pganalyze collector 
+running somewhere with access to your PostgreSQL instances once you have finished setting up the Enterprise Server Kubernetes deployment.
+ 
+<ToC items={props.toc} />
+
+## Installation steps
+
+### Pre-requisites
+
+* Provision a standalone Postgres database to store pganalyze statistics information
```

Comment:
```suggestion
* A standalone Postgres database to store pganalyze statistics information
```
or some other way to have parallel syntax for this and the next bullet point so it flows better